applications/asset_tracker/README.rst

```diff
@@ -53,6 +53,39 @@ You can also send AT commands from the **Terminal** card on nRF Cloud when the d
 
 By default, the asset tracker supports firmware updates through :ref:`lib_aws_fota`.
 
+.. _power_opt:
+
+Power optimization
+==================
+
+The Asset Tracker can run in three power modes that are configured in the Kconfig file of the application.
+Currently the dynamic switching between eDRX and PSM during run-time is only supported on the nRF9160 DK.
+However, both nRF9160 DK and Thingy:91 supports build-time configuration for enabling eDRX, PSM or both.
+
+.. note::
+   Not all cellular network providers support these modes, and the granted parameters can vary between networks.
+   Network operators can also limit the availability of power saving features for roaming users.
+   SIM card subscription also affects the availability of the cellular IoT power saving features.
+
+
+Demo mode
+	This is the default setting.
+	In this mode, the device maintains a continuous cellular link and can receive data at all times.
+	To enable this mode, set ``CONFIG_POWER_OPTIMIZATION_ENABLE=n``.
+
+Request eDRX mode
+	In this mode, the device requests the eDRX feature from the cellular network to save power.
+	In this mode, the device maintains a continuous cellular link.
```

Comment:
In this mode, the device maintains... -> The device maintains...
(Two consecutive sentences starting with "In this mode")